# Message that transports user commands. 

std_msgs/Header header
# manual activates / deactivates the high_level controller
bool master_on
# update parameters on all vehicles
bool update_param

# This allows the low level controller to be active
#
#CarCommand bbb0
#CarCommand bbb1
#CarCommand bbb2
#CarCommand bbb3
#CarCommand bbb4
#CarCommand bbb5
#CarCommand bbb6
#CarCommand bbb7
#CarCommand bbb8
#CarCommand bbb9
